WalletWasabi.Fluent/ViewModels/SearchBar/SearchBarViewModel.cs

```diff
@@ -54,7 +49,7 @@ public SearchBarViewModel(IObservable<IChangeSet<ISearchItem, ComposedKey>> item
 
 	private void ClearAndHideSearchList()
 	{
-		IsSearchListVisible = false;
 		SearchText = "";
+		ApplicationHelper.MainWindow?.Focus();
```

Comment:
Looks like we're breaking MVVM here? 

Also, calling this method from a background thread will cause a crash. It is risky to keep it this way.

I would rather not expose UI elements to ViewModel code, if possible.

Reply:
You're right. It should be fine now.